report/assets/styles.css

```diff
@@ -1820,20 +1786,23 @@
 
 
 .lh-meta__items {
+  --meta-icon-size: calc(var(--report-icon-size) * 0.667);
```

Comment:
Does this need to be full width? I kinda liked it when it was narrower + centered.